ui/src/components/header/ProjectMenu.vue

```diff
@@ -18,11 +18,11 @@
 <template>
   <span class="header-notice-opener">
     <a-select
+      v-if="showProjectMenu"
```

Comment:
I think just like this instead of creating a new variable
```suggestion
      v-if="!isDisabled()"
```

Reply:
+1